cluster/gce/util.sh

```diff
@@ -480,6 +480,10 @@ function create-node-template() {
   if [[ "${PREEMPTIBLE_NODE}" == "true" ]]; then
     preemptible_minions="--preemptible --maintenance-policy TERMINATE"
   fi
+  local local_ssds=""
+  for i in ${NODE_LOCAL_SSDS}; do
```

Comment:
Maybe it would be better if you also check for the existance of the variable too.